management/creative-template.yaml

```diff
@@ -99,6 +99,9 @@ paths:
                       Default:
                         type: object
                         nullable: true
+                      AdQuery:
+                        type: boolean
+                        default: false
```

Comment:
Since this isn't a required field, it should have `nullable: true` and no default.

We can only safely add default on non-optional fields in the API (otherwise we could create unexpected behavior).

Reply:
Updated